packages/hyperion-autologging/src/ALElementValuePublisher.ts

```diff
@@ -96,9 +96,9 @@ export function publish(options: InitOptions): void {
     }
 
     return tryQuery(
-      'input[type=radio][checked], input[type=checkbox][checked], select:has(option[selected])'
+      'input[type=radio][checked], input[type=checkbox], select:has(option[selected])'
```

Comment:
this might potentially worsen the performance for existing code as well. 
to be safe, you should return the old string when the flag is disabled, and have a separate one for flag enabled. 
Otherwise, even when flag is false, we do select all inputs and iterate over them.